src/api/entity.ts

```diff
@@ -103,7 +104,10 @@ export class Erc20TokensApi {
             }).then(count => {
                 const indexes = new Array(count).fill(0).map((_, i) => i)
 
-                return Promise.all(indexes.map(idx => CensusErc20Api.getTokenAddressAt(idx, gateway)))
+                // TODO Promise.allSettled is the correct one, should be used when target = ES2020 is fixed
+                return allSettled(indexes.map(idx => CensusErc20Api.getTokenAddressAt(idx, gateway)))
+            }).then((results: Array<{"value": any, "status": string}>) => {
```

Comment:
- The type of the results variable may be inferred. No need to define it (in theory)
- If we are returning the value of all results, why not filtering only the successful ones?

```ts
return results.filter(item => item.status == "fulfilled").map(item => item.value)

Reply:
I had to type the `map`:

```ts
return results
                    .filter(item => item.status === "fulfilled")
                    .map((item: {status: string, value: any}) => item.value)
```